syntax = "proto3";

package payment;

service PaymentService {
    rpc ProcessPayment(PaymentRequest) returns (PaymentResponse);
}

message PaymentRequest {
    string order_id = 1;
    double amount = 2;
}

message PaymentResponse {
    string payment_id = 1;
    string status = 2;
    string message = 3;
}
